firebase-firestore/api.txt

```diff
@@ -19,13 +19,39 @@ package com.google.firebase {
 
 package com.google.firebase.firestore {
 
+  public abstract class AggregateField {
+    method @NonNull public static com.google.firebase.firestore.AggregateField.AverageAggregateField average(@NonNull String);
+    method @NonNull public static com.google.firebase.firestore.AggregateField.AverageAggregateField average(@NonNull com.google.firebase.firestore.FieldPath);
+    method @NonNull public static com.google.firebase.firestore.AggregateField.CountAggregateField count();
+    method @NonNull @RestrictTo(androidx.annotation.RestrictTo.Scope.LIBRARY) public String getAlias();
+    method @NonNull @RestrictTo(androidx.annotation.RestrictTo.Scope.LIBRARY) public String getFieldPath();
+    method @NonNull @RestrictTo(androidx.annotation.RestrictTo.Scope.LIBRARY) public String getOperator();
+    method @NonNull public static com.google.firebase.firestore.AggregateField.SumAggregateField sum(@NonNull String);
+    method @NonNull public static com.google.firebase.firestore.AggregateField.SumAggregateField sum(@NonNull com.google.firebase.firestore.FieldPath);
+  }
+
+  public static class AggregateField.AverageAggregateField extends com.google.firebase.firestore.AggregateField {
+  }
+
+  public static class AggregateField.CountAggregateField extends com.google.firebase.firestore.AggregateField {
+  }
+
+  public static class AggregateField.SumAggregateField extends com.google.firebase.firestore.AggregateField {
+  }
+
   public class AggregateQuery {
     method @NonNull public com.google.android.gms.tasks.Task<com.google.firebase.firestore.AggregateQuerySnapshot> get(@NonNull com.google.firebase.firestore.AggregateSource);
+    method @NonNull public java.util.List<com.google.firebase.firestore.AggregateField> getAggregateFields();
```

Comment:
We didn't include this in the public API proposal, so we should consider adding `@RestrictTo(androidx.annotation.RestrictTo.Scope.LIBRARY)`